Language/Classes/Instance_Variables/covariant_A02_t02.dart

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion It is possible for the declaration of an instance variable to
+/// include the modifier covariant. The effect of this is that the formal
+/// parameter of the corresponding implicitly induced setter is considered to be
+/// covariant-by-declaration
+///
+/// @description Checks that if an instance variable can be declared as a
+/// covariant
+/// @author sgrekhov22@gmail.com
+
+import "../../../Utils/expect.dart";
+
+class A {
+  var v = 1 as num;
+}
+
+class C extends A {
+  covariant var v = 2;
```

Comment:
This one should have declared type `num` because of override inference, which means that the implicitly induced setter isn't using the extra freedom of a covariant parameter.

That's fine, we need to cover that case as well, but perhaps mention it in a comment?

Reply:
A comment added